docs/site/Server.md

```diff
@@ -58,6 +60,37 @@ export class HelloWorldApp extends RestApplication {
 
 ## Configuration
 
+### Enable HTTPS
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the crendential details.
```

Comment:
typo: `crendential` -> `credential` and I suggest we say `credentials` instead of `credential details`